tensorflow/compiler/xla/service/gpu/cudnn_fused_conv_rewriter.cc

```diff
@@ -149,6 +149,14 @@ absl::optional<ConvWithRelu> FindConvWithRelu(HloInstruction* instr) {
     return absl::nullopt;
   }
 
+  // In order to map to cudnnConvolutionBiasAcitvationForward for integer
```

Comment:
Today, this check can be replaced with an assert, since cudnn_conv_rewriter won't allow int8-to-int8 convolution go through.  Writing code in the current shape to avoid the assumption/dependency about cudnn_conv_rewriter.